pkg/crawler/crawler.go

```diff
@@ -63,6 +63,9 @@ func NewCrawler(opt Option) Crawler {
 
 func (c *Crawler) Crawl(ctx context.Context) error {
 	log.Println("Crawl maven repository and save indexes")
+	ctx, ctxCancelFunc := context.WithCancel(ctx)
```

Comment:
```suggestion
	ctx, cancel := context.WithCancel(ctx)
```

nit: I think `cancel` is used conventionally.
https://go.dev/doc/database/cancel-operations

Reply:
Done - https://github.com/aquasecurity/trivy-java-db/pull/27/commits/8f9c98b18e0a3005eb971f19d617f71a6369ba32